test/test_helper.exs

```diff
@@ -2,6 +2,9 @@
 # Needs to be called before ExUnit.start
 {:ok, _} = Application.ensure_all_started(:ex_machina)
 
+# By default, exclude S3 tests
+# To run S3 tests, run `mix test --include s3`
+ExUnit.configure exclude: [:s3]
```

Comment:
I mean, I can add to CI now if you'd like. I definitely see what you're saying. Maybe let's try it this way and see if it becomes an issue for folks later. Then we can fix.